### PR TITLE
Adapt extension for Windows OS

### PR DESCRIPTION
**Pull request by uarlouski**
Supersedes #15

### CODE REVIEW

File: vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/loader/LocalJavaProjectLoader.java [hunk 1]
```diff
@@ -66,7 +65,7 @@ public Optional<IJavaProject> load(String uri, Map<Event, Consumer<String>> hand
     {
         return RuntimeWrapper.wrapMono(() ->
         {
-            File projectFolder = Paths.get(new URI(uri).getPath()).toFile();
+            File projectFolder = new File(new URI(uri).getPath());
```
Comment (valfirst) [inline vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/loader/LocalJavaProjectLoader.java:68]: which issue is solved here?

File: src/lib/equinox.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ export function launch(exec: string, address: AddressInfo, application: Applicat
     serverArgs.push('-configuration');
     serverArgs.push(`${application.applicationDir}/${bundle}`);
 
-    const workspace = resolve(application.storageDir.path, 'workspace');
+    const workspace = resolve(application.storageDir.fsPath, 'workspace');
```
Comment (valfirst) [inline src/lib/equinox.ts:38]: what's the difference?

File: vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/loader/LocalJavaProjectLoader.java (hunk 1 above)
Comment (uarlouski) [reply vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/loader/LocalJavaProjectLoader.java:68]: The uri string has form \d:\path1\path2\test-project, new File(..) normalizes the form

File: src/lib/equinox.ts (hunk 2 above)
Comment (uarlouski) [reply src/lib/equinox.ts:38]: the path returns / separated path, whereas fsPath returns correct path like c:\path\path

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**